bliss/conf/base_config.yaml

```diff
@@ -12,14 +12,13 @@ hydra:
 
 paths:
     root: ${oc.env:BLISS_HOME}
-    data: ${paths.root}/data
-    sdss: ${paths.data}/sdss
-    decals: ${paths.data}/decals
-    des: ${paths.data}/des
-    dc2: ${paths.data}/dc2
+
+    sdss: /data/scratch/sdss
+    decals: /data/scratch/decals
+    des: /data/scratch/des
+    dc2: /data/scratch/dc2local
+
     output: ${paths.root}/output
-    pretrained_models: ${paths.data}/pretrained_models
-    dc2_catalog: ${paths.root}/case_studies/dc2_cataloging/merged_catalog
```

Comment:
@YicunDuanUMich I put a copy of the merged catalog under `/data/scratch/dc2local`. I also made a smaller version of this file, for use during testing, which I placed in `tests/data/dc2`.

Reply:
get it